tst/com/amazon/corretto/crypto/provider/test/EvpSignatureSpecificTest.java

```diff
@@ -448,11 +449,14 @@ public void testRsaPSSBadParams() throws Exception {
             signature.setParameter(
                 new PSSParameterSpec("SHA-1", "MGF1", MGF1ParameterSpec.SHA1, 4096, 1)));
 
-    // Assert compatiblity with JCE on setting null parameter, BC doesn't throw.
+    // Assert compatibility with JCE for JDKs other than 10 on setting null parameter, BC doesn't
+    // throw.
     assertThrows(InvalidAlgorithmParameterException.class, () -> signature.setParameter(null));
-    assertThrows(
-        InvalidAlgorithmParameterException.class,
-        () -> Signature.getInstance("RSASSA-PSS", "SunRsaSign").setParameter(null));
+    if (getJavaVersion() != 10) {
```

Comment:
Nit: I'd add the PR description as a comment here on this line before merging. 

`JDK 10 doesn't support RSASSA-PSS for signature when using SunRsaSign provider. We skip the compatibility test for JDK 10.`